exist-core/src/main/java/org/exist/xquery/value/DoubleValue.java

```diff
@@ -39,7 +41,7 @@
     public static final DoubleValue NEGATIVE_INFINITY = new DoubleValue(Double.NEGATIVE_INFINITY);
     public static final DoubleValue NaN = new DoubleValue(Double.NaN);
 
-    private final double value;
+    final double value;
```

Comment:
package protected on purpose?

Reply:
I think it's needed in `#createComparisonWith(NumericValue)`, did you see otherwise?

Reply:
just checking